Geometry_Engine/Query/UVCount.cs

```diff
@@ -33,8 +33,7 @@ public static partial class Query
 
         public static List<int> UVCount(this NurbsSurface surf)
         {
-            List<int> degrees = surf.Degrees();
```

Comment:
If you keep the `Degrees` method, you can avoid changing this as well.

Reply:
I will implement the change in order to avoid using a deprecated method.